docs/recipes/WritingTests.md

```diff
@@ -131,11 +131,16 @@ function mockStore(getState, expectedActions, onLastAction) {
 
       dispatch(action) {
         const expectedAction = expectedActions.shift();
-        expect(action).toEqual(expectedAction);
-        if (onLastAction && !expectedActions.length) {
-          onLastAction();
+        
+        try {
+          expect(action).toEqual(expectedAction);
+          if (done && !expectedActions.length) {
```

Comment:
shouldn't this be `if (typeof done === 'function' && !expectedActions.length) {`?